app/models/phone.rb

```diff
@@ -37,31 +31,35 @@ def international(unsafe_phone)
       parse(unsafe_phone).international(false)
     end
 
-    def send_confirmation_sms(phone)
-      Rails.logger.info("Sending SMS to #{phone.number}")
+    def send_confirmation(phone, channel)
+      Rails.logger.info("Sending code to #{phone.number}")
 
-      send_sms(number: phone.number,
-               content: Barong::App.config.sms_content_template.gsub(/{{code}}/, phone.code))
+      send_code(number: phone.number, channel: channel)
     end
 
-    def send_sms(number:, content:)
-      from_phone = Barong::App.config.twilio_phone_number
+    def send_code(number:, channel:)
```

Comment:
please extact to separate function

client = Barong::App.config.twilio_client
service_sid = Barong::App.config.twilio_service_sid

client.verify.services(service_sid)